kernel/arch/dreamcast/kernel/timer.c

```diff
@@ -171,6 +171,22 @@ void timer_spin_sleep(int ms) {
     timer_stop(TMU1);
 }
 
+void timer_spin_sleep_ns(unsigned short ns) {
+    uint64_t timeout = timer_ns_gettime64() + ns;
```

Comment:
```suggestion
    const uint64_t timeout = timer_ns_gettime64() + ns;
```

Reply:
Ehrm, what's the point of using "const" here?

Reply:
Nothing, really, the compiler is smart enough. I'm just anal. If you don't really care, then I guess I'll back off. lol.